js/reject-feedback.js

```diff
@@ -0,0 +1,88 @@
+( function( $, $gp ) {
+	// eslint-disable-next-line no-undef
+	$( document ).ready(
+		function() {
+			var feedbackForm = '<details><summary class="feedback-summary">Give feedback</summary>' +
+			'<div id="feedback-form">' +
+			'<form>' +
+			'<h3 class="feedback-reason-title">Reason</h3>' +
+			'<ul class="feedback-reason-list">' +
+			'<li><label><input type="checkbox" name="feedback_reason" value="style" />Style Guide</label></li>' +
+			'<li><label><input type="checkbox" name="feedback_reason" value="grammar" />Grammar</label></li>' +
+			'<li><label><input type="checkbox" name="feedback_reason" value="branding" />Branding</label></li>' +
+			'<li><label><input type="checkbox" name="feedback_reason" value="glossary" />Glossary</label></li>' +
+			'<li><label><input type="checkbox" name="feedback_reason" value="punctuation" />Punctuation</label></li>' +
+			'<li><label><input type="checkbox" name="feedback_reason" value="typo" />Typo</label></li></ul>' +
+			'<div class="feedback-comment">' +
+				'<label>Comment </label>' +
+				'<textarea name="feedback_comment"></textarea>' +
+			'</div>' +
+			'</form>' +
+			'</div>' +
+			'</details>';
+
+			// Remove click event added to <summary> by wporg-gp-customizations plugin
+			$( $gp.editor.table ).off( 'click', 'summary' );
+
+			$( 'button.reject' ).closest( 'dl,div.status-actions' ).prepend( feedbackForm );
```

Comment:
I've changed the query here so that it works for both GlotPress core and translate.wordpress.org